partcad/src/partcad/shape_ai.py

```
@@ -37,8 +38,13 @@ async def get_summary_async(self, project=None):
             self.project_name,
             self.name,
         )
-        if not self.desc is None:
-            prompt += 'The design is accompanied by the following description: "%s". ' % self.desc
+
+        if self.desc is not None:
+            prompt += "The design is accompanied by the description:\n```%s```\n" % self.desc
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)